.gitignore

```diff
@@ -43,6 +43,8 @@ vagrantconfig_local.yaml
 vagrant/manifests/classes/custom.pp
 node_modules
 .npm/
+static/css/node_lib/*
+static/js/node_lib/*
```

Comment:
As discussed on irc, since we don't prefix versions with ^ we shouldn't be taken by surprise with updates, however since there's no cast-iron guarantee around integrity, switching to yarn and verifying the lock file would be even better. npm-shrinkwrap would be another option too.